generator/src/test/java/com/scottlogic/deg/generator/cucumber/ContainingRegex.feature

```diff
@@ -136,7 +136,6 @@ Scenario: Running a 'containingRegex' request that includes special characters (
        | "け" |
        | "げ" |
 
-@ignore
```

Comment:
Shouldn't this test be left as ignored due to our restriction over plane-0 of unicode characters?